source/frontend/src/features/mapSideBar/lease/tabs/consultations/detail/ConsultationListView.tsx

```diff
@@ -235,3 +237,67 @@ const StyledIconWrapper = styled.div`
   justify-content: center;
   align-items: center;
 `;
+
+const StyledConsultationHeader = styled(Row)`
+  svg.info {
+    color: ${props => props.theme.bcTokens.surfaceColorBackgroundDarkBlue};
+  }
+  svg.error {
+    color: ${props => props.theme.bcTokens.typographyColorDanger};
+  }
+  svg.ok {
+    color: ${props => props.theme.css.completedColor};
+  }
+  svg {
+    vertical-align: baseline;
+  }
+`;
+
+const getOutcomeIcon = (consultations: ApiGen_Concepts_ConsultationLease[]) => {
+  if (consultations.length === 0) {
+    return null;
+  }
+
+  if (
+    consultations.find(c =>
+      [
+        ApiGen_CodeTypes_ConsultationOutcomeTypes.APPRDENIED.toString(),
+        ApiGen_CodeTypes_ConsultationOutcomeTypes.CONSDISCONT.toString(),
+      ].includes(c?.consultationOutcomeTypeCode?.id),
+    )
+  ) {
+    return (
+      <TooltipWrapper
+        tooltipId="error-outcome-tooltip"
+        tooltip="All approvals granted and/or consultations completed"
```

Comment:
please check the tooltips - I think they are not matching icon state. Doesn't look right that the error icon shows "all approvals granted..."